src/plugins/core/plugin_minify_html_literals.ts

```diff
@@ -0,0 +1,33 @@
+import { minifyHTMLLiterals } from 'minify-html-literals';
+import { Context } from '../../core/Context';
+
+export function pluginMinifyHtmlLiterals(...options: any) {
+  const useroptions = options;
+  return (ctx: Context) => {
+    ctx.ict.on('assemble_fast_analysis', props => {
```

Comment:
I also think it should be at `bundle_resolve_module` level, `assemble_fast_analysis` is a bit too early

Reply:
ok, I really dont know what comes first and last atm.
So will just update like you say :-)